_includes/header.html

```diff
@@ -21,7 +21,7 @@
         <div class="trigger">
           {% for path in page_paths %}
             {% assign my_page = site.pages | where: "path", path | first %}
-            {% if my_page.title %}
+            {% if my_page.title and my_page.exclude_from_nav == true %}
```

Comment:
Should be,
```ruby 
and my_page.exclude_from_nav != true
```
Thanks for sticking with this.  I know from experience getting boolean logic right can be a pain :-)